build.gradle.kts

```diff
@@ -294,7 +294,6 @@ tasks.register("javaPreCommit") {
   dependsOn(":sdks:java:fn-execution:build")
   dependsOn(":sdks:java:harness:build")
   dependsOn(":sdks:java:harness:jmh:build")
-  dependsOn(":sdks:java:io:azure:build")
   dependsOn(":sdks:java:io:bigquery-io-perf-tests:build")
   dependsOn(":sdks:java:io:common:build")
   dependsOn(":sdks:java:io:contextualtextio:build")
```

Comment:
there are still a couple of IO tests triggered in the large java precommit. Checked that these are not belong to specific IO PreCommit. We should move them into other tasks.

Reply:
this kind of bookkeeping is desirable because for now, if adding an IO connector, there is too much places to configure the test tasks and its possible to miss some of them (already happened)